docs/docs/guides/installing-the-cli.md

```diff
@@ -85,23 +85,11 @@ The Kurtosis CLI cannot be installed directly on Windows. Windows users are enco
 
 </Tabs>
 
-III. Add tab completion
+III. Add command-line completion
 -----------------------------
-[Kurtosis supports tab completion][installing-tab-completion], even for dynamic values like enclave names. We strongly recommend installing it for the best Kurtosis experience.
+[Kurtosis supports command-line completion][installing-command-line-completion], even for dynamic values like enclave names. We recommend installing it for the best Kurtosis experience.
 
-IV. Configure analytics
-------------------------
-By default, Kurtosis reports anonymized, obfuscated analytics which [we use only to improve the product][metrics-philosophy]. 
-
-If you'd like to [disable analytics][analytics-disable], you can run:
-
-```bash
-kurtosis analytics disable
-```
-
-To read more about why and how we collect product analytics metrics, [go here][metrics-philosophy].
-
-V. Run the quickstart
+IV. Run the quickstart
```

Comment:
```suggestion
Try the Quickstart
------------------
```

I like how this is the end of the installation guide and provides for a smooth ramp to get to the quickstart, but would remove it from the "required steps to install the CLI", since this guide is for installing the CLI alone.